hack/gen-release.sh

```diff
@@ -24,8 +24,8 @@ done
 echo "Prepare release for version $1"
 
 # Update release file
-RELEASE_FILE=release/RELEASE
-echo -e "version: $1" > $RELEASE_FILE
+RELEASE_FILE=RELEASE
+sed -i '' "s/tag:.*/tag: $1/" $RELEASE_FILE
```

Comment:
I know we don't have a convention for this but just think it's better to unify with other places we cut this value, so
```bash
head -n 1 RELEASE | cut -d ' ' -f 2
```
(as we did in Makefile)

Reply:
I see. Let me replace it with this 🙌

Reply:
@khanhtc1202 
Oops, sorry let me check one point.
This script updates the tag in the RELEASE file hence I think this args will be required.

Reply:
ah got that, it's my miss 🙏